security/voters.rst

```diff
@@ -35,17 +35,28 @@ The Voter Interface
 -------------------
 
 A custom voter needs to implement
-:class:`Symfony\\Component\\Security\\Core\\Authorization\\Voter\\VoterInterface`
+:class:`Symfony\\Component\\Security\\Core\\Authorization\\Voter\\VoterInterface`,
+:class:`Symfony\\Component\\Security\\Core\\Authorization\\Voter\\CacheableVoterInterface`
```

Comment:
```suggestion
:class:`Symfony\\Component\\Security\\Core\\Authorization\\Voter\\VoterInterface` or
:class:`Symfony\\Component\\Security\\Core\\Authorization\\Voter\\CacheableVoterInterface`
```

I'm not sure about why Voter implements both interfaces explicitly but since CacheableVoterInterface extends VoterInterface perhaps the documentation can mention that only one is needed ?